src/Controller/GalleryController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: wilder
+ * Date: 19/10/18
+ * Time: 09:44
+ */
+
+namespace Controller;
+
+use Model\GalleryManager;
+
+class GalleryController extends AbstractController
+{
+    /**
+     * @return string
+     * @throws \Twig_Error_Loader
+     * @throws \Twig_Error_Runtime
+     * @throws \Twig_Error_Syntax
+     */
+    public function gallery()
+    {
+        $galleryManager = new GalleryManager($this->getPdo());
+        $image = $galleryManager->selectAll();
```

Comment:
tu récupères une seule image ?

Reply:
On code en anglais.

Reply:
Message received. 
This correction is done. 
(i'm training my poor english).